examples/tutorials/aiida_setup/setup-external-computer.ipynb

```diff
@@ -0,0 +1,247 @@
+    "For more information on these settings, refer to the [AiiDA documentation](https://aiida.readthedocs.io/projects/aiida-core/en/stable/howto/run_codes.html#computer-setup). <br>\n",
+    "**Note:** We are loading the module we want in `prepend_text`. As we will be needing Quantum Espresso to load the code in the later steps we will load it now, otherwise you can leave it empty (`prepend_text: ''`)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "65057015",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile ~/computer.yaml\n",
+    "label: [scarf]\n",
+    "description: [https://www.scarf.rl.ac.uk/index.html]\n",
+    "hostname: [ui1.scarf.rl.ac.uk]\n",
+    "transport: core.ssh\n",
+    "scheduler: core.slurm\n",
+    "shebang: '#!/bin/bash'\n",
+    "work_dir: [/work4/scd/scarf/aiida]\n",
+    "mpirun_command: srun -u -n {tot_num_mpiprocs}\n",
+    "mpiprocs_per_machine: 32\n",
+    "use_double_quotes: False\n",
+    "prepend_text: |\n",
+    "  module load amd-modules\n",
+    "  module load QuantumESPRESSO/7.2-foss-2023a\n",
+    "append_text: ''"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c7b34fe6",
+   "metadata": {},
+   "source": [
+    "Run the following commands. First we set up the computer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "811e10d3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! verdi computer setup -n --config ~/computer.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "963d43fa",
+   "metadata": {},
+   "source": [
+    "Then we can configure the computer for the ssh transport, make sure to change the settings in square brackets with your settings. The [AiiDA documentation](https://aiida.readthedocs.io/projects/aiida-core/en/stable/howto/run_codes.html#computer-connection-configuration) goes over how to configure a computer in more detail."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eaf2e067",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! verdi computer configure core.ssh [scarf] -n --username [scarf] --key-filename ['/path/to/ssh/key'] --key-policy AutoAddPolicy --safe-interval 30.0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a8fccf0e",
+   "metadata": {},
+   "source": [
+    "Once the computer is set up, you can test the connection"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ce2132c3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! verdi computer test [scarf]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eec65a5b",
+   "metadata": {},
+   "source": [
+    "If you have a SSH config file setup in ~/.ssh/config, you can directly set the transport layer by following the steps in the [AiiDA documentation](https://aiida.readthedocs.io/projects/aiida-core/en/stable/howto/ssh.html#ssh-configuration)."
```

Comment:
I don't think I'd put this right at the end, otherwise no one is going to see it until they've also run the cells above that may well not be necessary. I'd probably include it in the markdown cell before `verdi computer configure core.ssh`